packages/animations/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: animations
 description: Fancy pre-built animations that can easily be integrated into any Flutter application.
-version: 2.0.0
+version: 2.0.1
```

Comment:
This needs a corresponding CHANGELOG update.

Reply:
Done.